From 954e4768ff5cfcc55974a3a3e684a458ebe7fec2 Mon Sep 17 00:00:00 2001
From: dartvader316 <dartvader316-dev@pm.me>
Date: Thu, 3 Nov 2022 18:43:14 +0300
Subject: [PATCH] Revert "partially Revert "usb: musb: Set the DT node on the
 child device""

 espresso10 musb fix

This reverts commit 239071064732bc4a30308cbba11014aa1aab550a.
---
 drivers/usb/musb/omap2430.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/usb/musb/omap2430.c b/drivers/usb/musb/omap2430.c
index f571a65ae6ee..4a963cfa385b 100644
--- a/drivers/usb/musb/omap2430.c
+++ b/drivers/usb/musb/omap2430.c
@@ -327,6 +327,7 @@ static int omap2430_probe(struct platform_device *pdev)
 	musb->dev.parent		= &pdev->dev;
 	musb->dev.dma_mask		= &omap2430_dmamask;
 	musb->dev.coherent_dma_mask	= omap2430_dmamask;
+	device_set_of_node_from_dev(&musb->dev, &pdev->dev);
 
 	glue->dev			= &pdev->dev;
 	glue->musb			= musb;
-- 
GitLab

